rfcs/0003-timestamps-and-ttl.md

```diff
@@ -211,8 +177,11 @@ table SsTableInfo {
     // the current encoding version of the data block
     format_version: uint;
     
-    // the metadata features that are encoded with each row, in order that they are encoded
+    // the metadata features that are encoded with each row, in declaration order in MetaFeatures
     meta_features: [MetaFeatures];
+    
+    // the time at which this SST was created
+    creation_timestamp: long;
```

Comment:
Not sure whether we want to go full 😵‍💫 but this could be `creation_clock_tick` or something if we want to capture non-time-based clocks.

Reply:
😵 haha I'm in support of keeping it time-focused, people can just have their own definition of time that's application specific